### PR TITLE
increase UnitTimeoutInactive inactivityTimeout

### PR DESCRIPTION
**Pull request by caolanm**
fails for me locally with --enable-debug 'CFLAGS=-O0 -ggdb' 'CXXFLAGS=-O0 -ggdb'

Change-Id: I8cd0cb1d187bf3adab1f81345c1cd83b5180eee4

* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] I have run `make prettier-write` and formatted the code.
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by caolanm** [comment]
This is fairly edge case I guess, but it looks harmless to me to increase this.

File: test/UnitTimeoutInactive.cpp [hunk 1]
```diff
@@ -37,7 +37,7 @@ class UnitTimeoutInactivity : public UnitTimeoutBase0
     void configure(Poco::Util::LayeredConfiguration& /* config */) override
     {
         // net::Defaults.inactivityTimeout = std::chrono::seconds(3600);
-        net::Defaults.inactivityTimeout = std::chrono::milliseconds(100);
+        net::Defaults.inactivityTimeout = std::chrono::milliseconds(150);
```
Comment (vmiklos) [inline test/UnitTimeoutInactive.cpp:40]: Aha, the default is 3600 seconds and we decrease this drastically to have a faster test, but then with 100ms we turn a stream socket into an inactive one so fast that it happens when no timeout should happen in a debug build, and increasing that decreased value makes the test more stable, while still not having to wait for 1h for the inactivity to happen. Makes sense after reading this about 3 times. :-)

**Review by vmiklos — APPROVED** [review]
